munimap_transport/munimap_transport/static/js/transport-controller.js

```diff
@@ -157,7 +157,7 @@ angular.module('munimapBase')
                     stationPointLayer.changeUrl(stationPointLayerURL + layer);
                     updateTimetableDate();
                     $scope.stationPopupLayers = [stationsLayer, stationPointLayer];
-                    $scope.excludePopupLayers = $scope.popupLayers.concat($scope.stationPopupLayers);
+                    $scope.excludePopupLayers.push(...$scope.popupLayers, ...$scope.stationPopupLayers);
```

Comment:
popupLayers should already be included
```suggestion
                    $scope.excludePopupLayers.push(...$scope.stationPopupLayers);
```

Reply:
See https://github.com/stadt-bielefeld/bielefeldGEOCLIENT/blob/a87d8b8e3bc5d48f7aa996002bb051a77d84acac/munimap/static/js/base-controller.js#L79-L86